go-controller/pkg/ovn/ovn.go

```diff
@@ -469,19 +474,53 @@ func (oc *Controller) checkAndDeleteRetryPod(uid types.UID) bool {
 	return false
 }
 
-// addRetryPod tracks a failed pod to retry later
-func (oc *Controller) addRetryPod(pod *kapi.Pod) {
+// checkAndSkipRetryPod sets a specific entry from the map to be ignored for subsequent retries
+// if it existed, returns true
+func (oc *Controller) checkAndSkipRetryPod(uid types.UID) bool {
+	oc.retryPodsLock.Lock()
+	defer oc.retryPodsLock.Unlock()
+	if entry, ok := oc.retryPods[uid]; ok {
+		entry.ignore = true
+		oc.retryPods[uid] = entry
```

Comment:
Maybe we should make retryPods hold pointers to retryEntry; but that can be later.

Reply:
if you like I can do in this PR. Are there any other comments besides this?

Reply:
@trozet Nope.